diktat-rules/src/main/resources/diktat-analysis.yml

```diff
@@ -12,6 +12,8 @@
 # Checks that the Class/Enum/Interface name does not match Pascal case
 - name: CLASS_NAME_INCORRECT
   enabled: true
+  # all code blocks with MyAnnotation will be ignored and not checked
+  ignoreAnnotated: [ MyAnnotation ]
```

Comment:
This file will be bundled into the jar, is it worth adding this here and not into examples?

Reply:
I see that users copy-paste this one 🤦